tests/scalems_test_cfg.json

```diff
@@ -0,0 +1,20 @@
+
+{
+    "cpn"         :  4,
+    "gpn"         :  0,
+    "n_masters"   :  1,
+    "n_workers"   :  1,
+
+    "master"      : "scalems_test_master.py",
+    "worker"      : "scalems_test_worker.py",
+
+    "pilot_descr" : {
+        "runtime"     : 30,
+        "resource"    : "local.localhost"
+    },
+    "worker_descr": {
+        "executable"  : "`which python3`",
```

Comment:
In the long run we need to get rid of the backticks - not all launch methods interpret shell.

Reply:
Should we open a SCALE-MS issue or a radical.pilot issue? Or should we build this file at runtime using `sys.executable`?

Reply:
> Should we open a SCALE-MS issue or a radical.pilot issue?
I don't know.  `mpirun`/`mpiexec` are our work horses, and we don't see this problem in any other context, which is very surprising.  But also, the master and worker startup are about to get a tighter integration with the agent anyway, so this specific problem will disappear for unrelated reasons...

Reply:
Does the upcoming change mean that `worker_descr['executable']` is going away, or just that the absolute path will not be necessary? In other words, do we consider this resolved or is there a new action item that depends on an existing issue?

Reply:
The worker should not have much of a description in the first place.  Instead, the RP will provide an API to register a worker class or script.  Same for the master BTW.

Reply:
So `worker_descr` is going away, so there is no "to do", right?

Reply:
It might take a while, but that is correct.